elasticsearch/prep-install.origin

```diff
@@ -7,6 +7,18 @@ if [ -z "${ES_CLOUD_K8S_URL:-}" ] ; then
     ES_CLOUD_K8S_URL=io.fabric8/elasticsearch-cloud-kubernetes/${ES_CLOUD_K8S_VER}
 fi
 if [ -z "${OSE_ES_URL:-}" ] ; then
-    OSE_ES_URL=io.fabric8.elasticsearch/openshift-elasticsearch-plugin/${OSE_ES_VER}
+    MANPATH="" source /opt/rh/rh-maven33/enable
+    set +e
+    mvn dependency:get -Dartifact=io.fabric8.elasticsearch:openshift-elasticsearch-plugin:${OSE_ES_VER}
+    res=$?
+    set -e
+    if [ 0 -eq $res ]; then
+      OSE_ES_URL=io.fabric8.elasticsearch/openshift-elasticsearch-plugin/${OSE_ES_VER}
+    else
+      pushd /tmp/lib/openshift-elasticsearch-plugin
+        mvn clean verify -DskipTests
```

Comment:
This also spews a lot of output

Reply:
i believe this replicates a lot of what 'get' does.